src/Symfony/Component/Security/Http/Firewall/ContextListener.php

```diff
@@ -209,6 +210,14 @@ protected function refreshUser(TokenInterface $token)
             }
         }
 
+        if (isset($deauthenticationContext)) {
```

Comment:
we prefer assigning to `null` by default, checking for `if ($deauthenticationContext)` instead of `isset`